SQL/Archive/2017-12-15_media_add_language_to_UploadFile

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE media ADD COLUMN  language_id INT UNSIGNED DEFAULT NULL;
```

Comment:
this filename is missing the .sql extension

Reply:
solved